test/boost/bloom_filter_test.cc

```diff
@@ -89,7 +94,7 @@ SEASTAR_TEST_CASE(test_sstable_manager_auto_reclaim_and_reload_of_bloom_filter)
 
         // Test auto reload - disposing sst3 should trigger reload of the
         // smallest filter in the reclaimed list, which is sst1's bloom filter.
-        shared_sstable::dispose(sst3.release().release());
+        dispose_and_stop_tracking_bf_memory(std::move(sst3), sst_mgr);
```

Comment:
This is slightly icky - from a very neutral dispose() the name now encodes some implementation details. But being explicit is also good, so okay.

Reply:
This is just a local wrapper in the test file. The `dispose()` in sstable class remains untouched.